scenarioo-client/app/types/backend-types.ts

```diff
@@ -0,0 +1,278 @@
+/* tslint:disable */
+
+export interface IConfiguration {
```

Comment:
I think it would be great to use this type definitions already in the services. Otherwise they are not of much use... ☺️

Reply:
I'll do that in the other PRs. I want to have this change in develop first for everybody to use. So I can merge it into the other branches.

Reply:
Done